doc/connectivity/functional_connectomes.rst

```diff
@@ -208,10 +208,14 @@ with MNI coordinates for each region (see for instance example:
     >>> from nilearn import datasets
     >>> atlas_filename = datasets.fetch_msdl_atlas().maps
 
-For another atlas this information can be computed for each region with
-the :func:`nilearn.plotting.find_xyz_cut_coords` function
-(see example:
-:ref:`example_connectivity_plot_multi_subject_connectome.py`)::
+For atlases without readily available label coordinates, center coordinates
+can be computed for each region on hard parcellation or probabilistic atlases.
+
+  * For hard parcellation atlases (eg. :func:`datasets.fetch_atlas_destrieux_2009`),
+  use the :func:`nilearn.plotting.find_parcellation_atlas_cut_coords` function.
```

Comment:
The indentation of this line is wrong (same thing below): it needs to be indented with the same indentation as the 'F' of the 'For' above.